components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -50,8 +56,25 @@
 @Categorized(superCategory = ImproveSuperCategory.class, value = ReferenceDataCategory.class)
 public class RemoveDictionaryMatchesTransformer implements Transformer {
 
+    public static enum RemovedMatches implements HasName {
```

Comment:
Consider if we can find a better name for the enum. Consider if it can describe that it's a choice/option/setting/type.